src/utils/metrics.ts

```diff
@@ -46,20 +46,20 @@ export const blockHeight = new client.Gauge({
 });
 
 export const reorgDepth = new client.Gauge({
-  name: "watch_tower_reorg_depth",
-  help: "Depth of the current reorg",
+  name: "watch_tower_latest_reorg_depth",
+  help: "Depth of the last reorg",
```

Comment:
Suggest from a grammar side that the `help` uses the same tense, or we use something different completely such as:

`help: "Depth of the most recent reorg"`

Reply:
https://github.com/cowprotocol/watch-tower/pull/111